Assets/Scripts/Audio/MusicTrackManager.cs

```diff
@@ -63,6 +67,8 @@ private MusicTrack GetTrack(MusicType type)
         {
             case MusicType.BATTLE:
                 return battleTheme;
+            case MusicType.FANFARE:
+                return constructionFanfare;
```

Comment:
Please rename either the enum or the member (preferrably the former in case we want other fanfares) so they have the same name.